importer/api/__init__.py

```diff
@@ -142,6 +144,7 @@ def import_children(course, element, parent):
         content_xml=etree.tostring(element),
         mpath=mpath,
     )
+    target = "/static/"
```

Comment:
It needs to be used twice; once to check for it, and again to strip it off of the path.

Before, it was like this:

```python
if val.startswith("/static/"):
    # more code here
    asset=course_asset_basepath(course, val[8:])
```

I thought it was nicer to make it explicit as shown below (`val[len(target)]`. This makes it clear that the `path` stripping is based on the length of `/static/`.